### ISSUE TITLE
wasm-ld compilation error on simple token 

### ISSUE THREAD

**Issue by ValeryDubrava** [EOSIO/eosio.wasmsdk#54]
OS: Cenots7
Branch: develop
```
/usr/local/eosio.wasmsdk/bin/eosio-cpp -isystem=/home/valery/opt/boost/include -o eosio.token/eosio.token.wast eosio.token.cpp
/usr/local/eosio.wasmsdk/bin/wasm-ld: error: unable to find library -lc++
/usr/local/eosio.wasmsdk/bin/wasm-ld: error: unable to find library -lc
/usr/local/eosio.wasmsdk/bin/wasm-ld: error: unable to find library -leosio
```
Possible, I have to specify path for shared libraries, but I'm not sure how can I do it.

**Comment by ValeryDubrava** [comment]
I tried to add -l=/usr/local/eosio.wasmsdk/lib64/libeosio.a, but received new one error:
`/usr/local/eosio.wasmsdk/bin/wasm-ld: error: unable to find library -l/usr/local/eosio.wasmsdk/lib64/libeosio.a`

**Comment by ValeryDubrava** [comment]
-L=/usr/local/eosio.wasmsdk/lib64 helps me

**Comment by larryk85** [comment]
resolved by PR #62
